Library/Formula/node.rb

```diff
@@ -55,42 +56,50 @@ def install
     system "./configure", *args
     system "make", "install"
 
-    resource("npm").stage libexec/"npm" if build.with? "npm"
-  end
+    if build.with? "npm"
+      resource("npm").stage libexec/"npm_install"
 
-  def post_install
-    return if build.without? "npm"
+      # make sure npm can find node
+      ENV["PATH"] = "#{Formula["node"].prefix}/bin:#{ENV["PATH"]}"
+
+      # set log level temporarily for npm's `make install`
+      ENV["NPM_CONFIG_LOGLEVEL"] = "verbose"
+
+      (libexec/"npm_install").cd { system "./configure", "--prefix=#{libexec}/npm" }
+      (libexec/"npm_install").cd { system "make", "install" }
+
+      # FIXTHIS: Manpages aren't currently linking, which is frustrating.
+      rm_rf libexec/"npm/lib/node_modules/npm/share"
```

Comment:
I tried ` rm_rf ` but Brew wasn't interested in that either, frustratingly. I pretty seriously considered writing in a mini version of my ` Node removal script ` and including that until the next ` Node ` release, because that shells out without involving ` brew ` and theoretically, would do the job. Theoretically.